plugins/engine-configuration/pom.xml

```diff
@@ -17,6 +17,7 @@
 
   <properties>
     <pdi.version>9.3.0.0-SNAPSHOT</pdi.version>
+    <csrf.token.service.client.version>0.8.0</csrf.token.service.client.version>
```

Comment:
There already exists a property in the [pentaho parent pom](https://github.com/pentaho/maven-parent-poms/blob/master/pom.xml#L353) for this that we _must_ use — `hv-security-web.version`.
So this line should be removed.

Reply:
addressed the comments